Header header
uint8 vert=1
uint8 horz=0
uint8 flip_direction
